assets/marketplace/resources/itemlist.json

```diff
@@ -0,0 +1,69 @@
+{
```

Comment:
Can we remove these mock up files?

Reply:
Sure, this is a draft PR, I will do code clean up once PR is totally ready.